packages/aws-cdk/bin/cdk.ts

```diff
@@ -51,7 +51,7 @@ async function parseCommandLineArguments() {
     .command([ 'synthesize [STACKS..]', 'synth [STACKS..]' ], 'Synthesizes and prints the CloudFormation template for this stack', yargs => yargs
       .option('exclusively', { type: 'boolean', alias: 'e', desc: 'only deploy requested stacks, don\'t include dependencies' }))
     .command('bootstrap [ENVIRONMENTS..]', 'Deploys the CDK toolkit stack into an AWS environment', yargs => yargs
-      .option('toolkit-bucket-name', { type: 'string', alias: 'b', desc: 'The name of the CDK toolkit bucket', default: undefined })
+      .option('toolkit-bucket-name', { type: 'string', alias: ['b', 'bootstrap-bucket-name'], desc: 'The name of the CDK toolkit bucket', default: undefined })
```

Comment:
Sorry for being annoying, can you make bootstrap-bucket-name the primary?

Reply:
Sure, give me a second. I am fixing the multiple LambdaIntegrations PR currently

Reply:
Done!